src/vtk_calculator.py

```diff
@@ -132,17 +131,35 @@ def main():
                 "90th percentile": p90
             }, open(stat_file, "w"))
 
+    else:
+        calc.SetFunction(args.function)
+        logging.info("Evaluated \"{}\" on the mesh.".format(args.function))
```

Comment:
```suggestion
        logging.info("Evaluated the input function \"{}\" on the mesh.".format(args.function))
```
Maybe also add the data name here.